### DISCUSSION TITLE
Pass currentSite or other variables to system emails

### DISCUSSION THREAD

**Discussion by lenvanessen** [craftcms/cms#13330] [Ideas] [answered]
Hi There,

We run a multisite with different brands, and want to use some context to customize that layout of our e-mails. 
For instance, when a user uses the front-end password reset form, it would be great if we can access the site it was sent from, perhaps using some extra body params in the form itself?

This way, we'd be able to tailor the mails a bit more to each site.

**Comment by thupsi** [comment]
I might be overlooking something, but could you perhaps pass these params using hidden fields?

**Reply by lenvanessen** [reply]
@thupsi  they wont be passed to the template context of the email unforunately

**Reply by thupsi** [reply]
Ah, I see! Then I totally agree with your suggestion!

**Comment by brandonkelly** [comment, marked as answer]
`currentSite` and other global variables are available to your email template, just like front-end templates.
